pkg/controller/apiextensions/definition/reconciler.go

```diff
@@ -160,6 +160,14 @@ func WithCRDRenderer(c CRDRenderer) ReconcilerOption {
 	}
 }
 
+// WithClientApplicator specifies how the Reconciler should interact with the
+// Kubernetes API.
+func WithClientApplicator(ca resource.ClientApplicator) ReconcilerOption {
```

Comment:
Not related to the changes in this PR specifically, but could we potentially now use the updating applicator in this reconciler following https://github.com/crossplane/crossplane-runtime/pull/176?

Reply:
Possibly, but I'd prefer to worry about that in a future PR.